pkg/minikube/openshiftversions/openshift_versions.go

```diff
@@ -36,7 +36,7 @@ func PrintOpenShiftVersionsFromGitHub(output io.Writer) {
 func PrintOpenShiftVersions(output io.Writer) {
 	versions, err := getVersions()
 	if err != nil {
-		glog.Errorln(err)
```

Comment:
changing some logging. This caused problems when running test on Windows. Also glog is not the right logger here. For now fmt.Println which needs to be revisited once we look at an overall logging solution